moped-database/migrations/1698428471744_create_signage_line_component/up.sql

```diff
@@ -0,0 +1,35 @@
+-- reset coponent id sequence to max component ID
+SELECT
+    setval('moped_components_component_id_seq', (
+            SELECT
+                max(component_id)
+                FROM moped_components));
```

Comment:
while testing locally with prod data i discovered that our component ID sequence was at 128 although we only had 97 components in the DB. my theory is that i screwed up the sequence when we had that failed/aborted migration a few releases ago.

This was also a good reminder to add the ID sequence resetter to down migrations.